### PR TITLE
More calendar fixes

### PR DESCRIPTION
**Pull request by jancborchardt**
- clarify non-shared message, also add for shared via calendar
- fix share dropdown being hidden and hiding calendar which will be shared
- remove confusing note, instead only make it possible to edit calendar sharing settings in the calendar settings cc @te-online
- only append (group) indicator to groups, no need to put (user) after every user
- improve look of sharing tab
- remove 'can delete' permission, also resolve CSS 'delete' class conflict cc @DeepDiver1975 @georgehrke 

Please review @georgehrke @owncloud/designers – Georg, I’m not sure if I did the removal of »can delete« correctly.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Looks great :+1:

**Comment by te-online** [comment]
Cool :+1:

**Comment by jancborchardt** [comment]
Thanks for the reviews!

I’d like to have a last look from @DeepDiver1975 or @georgehrke or someone who can verify that I removed the »can delete« correctly.

File: js/calendar.js [hunk 1]
```diff
@@ -594,10 +596,9 @@ Calendar={
 						var permission = null;
 						if($(this).hasClass('update')) {
 							permission = OC.PERMISSION_UPDATE;
+							permission = OC.PERMISSION_DELETE;
```
Comment (DeepDiver1975) [inline js/calendar.js:599]: I guess it should be update and delete?

+= would help from my understanding

**Comment by DeepDiver1975** [comment]
:-1: it's loosing the checked 'can edit' state

**Comment by jancborchardt** [comment]
> it's loosing the checked 'can edit' state

Also happens with master, see https://github.com/owncloud/calendar/issues/217

**Comment by DeepDiver1975** [comment]
> Also happens with master, see #217

in this case :+1:

File: js/calendar.js (hunk 1 above)
Comment (jancborchardt) [inline js/calendar.js:599]: Oi, you merged, but what about this then? ;)

Comment (DeepDiver1975) [inline js/calendar.js:599]: this part has no influence at al to the behavior - this is the code piece which is broken and causes the check box not to be set properly

**Pull request merged**